README.md

```diff
@@ -140,38 +140,44 @@ images:
 
 Follow these instructions to set up the dependencies for the scripts used for visualization and creation of the dataset.
 
-```
-# Clone the repository
-git clone https://github.com/bit-bots/TORSO_21_dataset.git
-cd TORSO_21_dataset
+1. Install the package manager Poetry as described [here](https://python-poetry.org/docs/#installation). This prevents dependency conflicts and ensures that the correct versions of the dependencies are installed.
 
-# Install poetry
-pip3 install poetry --user
+2. Clone the repository:
 
-# Install dependencies
-poetry install --no-dev
+    ```shell
+    git clone https://github.com/bit-bots/TORSO_21_dataset.git
+    ```
 
-# Optionally install dependencies including those for the dataset creation
-poetry install
-```
+3. Move into the repository and install the dependencies
 
-### Usage
+    - without optional dependencies (for dataset creation):
 
-To run the tools you need to source the poetry environment in your shell.
+        ```shell
+        cd TORSO_21_dataset && poetry install --without=dev --no-root
+        ```
 
-```
-# Source the virtualenv
+    - with optional dependencies (for dataset creation):
+
+        ```shell
+        cd TORSO_21_dataset && poetry install --no-root
```

Comment:
Do we actually want to run this with `no-root`?
Currently, the directory structure is wrong to be an actual package, thus the installation of this root package fails.